packages-backend/loggers/src/create-access-logger.ts

```diff
@@ -0,0 +1,51 @@
+import type { RequestFilter } from 'express-winston';
+
+import expressWinston from 'express-winston';
+import winston from 'winston';
+import * as env from './env';
+import redactInsecureRequestHeaders from './utils/redact-insecure-request-headers';
+import jsonFormatter from './custom-formats/json';
+
+const shouldLog = !env.isTest || process.env.DEBUG === 'true';
+
+type LoggerOptions = {
+  ignoreUrls?: string[];
+};
+
+const defaultIgnoreUrls = ['/', '/health', '/favicon.ico'];
+const defaultFormat = winston.format.combine(winston.format.timestamp());
+
+// Inspired by https://github.com/bithavoc/express-winston/issues/62#issuecomment-396906056
+const requestFilter: RequestFilter = (req, propName) => {
+  if (propName === 'headers') {
+    return redactInsecureRequestHeaders(req);
+  }
+  return req[propName];
+};
+
+const createAccessLogger = (options: LoggerOptions = {}) => {
```

Comment:
This is a middleware I think. So to have it inline with others maybe it's `createAccessLoggerMiddleware(options)`.